source/comparisons.rst

```diff
@@ -10,7 +10,28 @@ Details and porting strategies follow.
 Unorderable Types
 ~~~~~~~~~~~~~~~~~
 
-XXX
+Strict approach to comparing in Python 3 makes it impossible to compare the
+different types of objects.
+
+While this example where we try to compare ``int`` and ``str`` in Python 2
+works (with bad result)::
+
+    >>> 2 < '2'
+    True
+
+In Python 3 not::
```

Comment:
You can put the whole sentence up here: In Python 3, it fails with a well described error message::